### PR TITLE
add url for all mooc_providers

### PR DESCRIPTION
**Pull request by rosabraatz**
add urls in seeds 
show url on settings page
link to provider from course detail page

### CODE REVIEW

**Comment by MrSerth** [comment]
Thanks for your PR. It's a great idea to add a link to the providers. 

Just some thoughts:
- Is it by intent to merge this to the master right away? Besides that, I would like to merge the dev to master very soon. 
- Could you please add a URL for open_mammoov in the seeds as well (only applied in the dev environment)?
- I would prefer to have a common ending for all URLs. Currently, you use some URLs ending with an back slash and other without. Right now, it's no problem but when we might use it in the future, we might run into trouble. 

I'll have another look later today.

File: app/views/users/_mooc_provider_settings.html.slim [hunk 1]
```diff
@@ -9,6 +9,7 @@ h2
           .provider
             a data-toggle="collapse" href="#collapse-#{provider[:id]}"
               img src = @provider_logos[provider[:logo_id]]
+              = " #{provider[:url]}"
```
Comment (MrSerth) [inline app/views/users/_mooc_provider_settings.html.slim]: Is there a simple solution to prevent the 'space' from being underlines when you hover it?

**Comment by MrSerth** [comment]
Do you think, we should make the provider logo on the course detail page a link as well?

**Comment by MrSerth** [comment]
I merged the dev to master so that my question regarding merging this PR to the master is not longer important.

**Comment by jprberlin** [comment]
:+1:

**Pull request merged**